tests/conftest.py

```
@@ -527,17 +527,24 @@ def pytest_collection_modifyitems(session, config, items):
                             f"Test: {item} it will be skipped because the OCS cluster"
                             " has not configured cluster-wide encryption with KMS"
                         )
                         items.remove(item)
-                except KeyError:
-                    log.warning(
-                        "Cluster is not yet installed. Skipping skipif_no_kms check."
+                else:
+                    # ODF not deployed, skip all KMS-requiring tests
+                    log.debug(
+                        f"Test: {item} will be skipped (ODF not deployed, "
+                        "skip_ocs_deployment or ocp_only_upgrade is set)"
```

Ruff (0.16.3)

[warning] Logging statement uses f-string

(G004)